File: app/src/main/java/kotleni/b0mb3r/services/Citilink.java
package kotleni.b0mb3r.services;

import okhttp3.HttpUrl;

public class Citilink extends ParamsService {

    public Citilink() {
        setUrl("https://www.citilink.ru/registration/confirm/phone/");
        setMethod(POST);
        setPhoneCode("7");
    }

    @Override
    public void buildParams(HttpUrl.Builder builder) {
        builder.addPathSegment("+" + getFormattedPhone() + "/");
    }
}

File: app/src/main/java/kotleni/b0mb3r/services/Citilink.kt
package kotleni.b0mb3r.services

import okhttp3.HttpUrl.Builder

class Citilink : ParamsService() {
    override fun buildParams(builder: Builder) {
        builder.addPathSegment("+$formattedPhone/")
    }

    init {
        setUrl("https://www.citilink.ru/registration/confirm/phone/")
        setMethod(POST)
        setPhoneCode("7")
    }
}